### ISSUE TITLE
media_player becomes unavailable after HA restarts

### ISSUE THREAD

**Issue by aaamoeder** [thomasloven/hass-browser_mod#509]
My Home Assistant version: 2022.12.8

**What I am doing:**
using new install of browser mod to use on kiosk

**What I expected to happen:**
Playing alerts on kiosk through media_player (which works fine after enabling browser mod)

**What happened instead:**
after HA restarts the media_player becomes unavailable.

**Minimal** steps to reproduce:
restart ha
try to play sound

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by lylavoie** [comment]
I'm having the same issue with the same version of HA (2022.12.8), and the latest browser mode release 2.2.0.  I haven't been able to work out the "magic" steps to get it to come back reliably.  Reloading the target browser (kiosk) don't reliably make it available, and toggling the "enabled" setting on browser mod config doesn't fix it either.

Most recently, it was not working, I turned on "debug" on the integration and went over the kiosk to toggle the browser mode enabled setting.  The screen was showing the "tap here" icon that is mentioned on the config page, and tapping it "fixed it."  However, it was still showing as "unavailable" until I called the play_media service, which worked.  

But, I have not seen these steps as reliable to fix it.  Breaking, with an HA restart (restarting the docker container) does seem to break it pretty reliably.
